src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  const selectTab = (tab) => {
+    if (tab.id !== selectedTab.id) {
+      onTabSelected(tab);
+    }
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
```

Comment:
Would be great to use destructuring for better readability

Reply:
Sorry. I don’t understand what I can destruct here!

Reply:
```suggestion
          {tabs.map(({ id, title }) => (
```

Reply:
I undestand. But for OnClick need all tab properties. And I must destruct all the properties Tab. In future  we can add properties for tabs and it crashed our project. Maybe i do not right , but is it my opinion.